iosApp/iosApp/ComponentViews/RouteCard/RouteCardList.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  RouteCardList.swift
+//  iosApp
+//
+//  Created by Melody Horn on 6/12/25.
+//  Copyright © 2025 MBTA. All rights reserved.
+//
+
+import Shared
+import SwiftUI
+
+struct RouteCardList<EmptyView: View>: View {
```

Comment:
This and `LoadingRouteCard` are helper components that already exists on Android. At some point, we may want to review our component trees and make sure they match between Android and iOS, because every time we have a component that exists in Android but not iOS or vice versa it’s a little bit of a headache.